composer.json

```diff
@@ -28,12 +28,12 @@
     "require-dev": {
         "phpunit/phpunit": "^7.5",
         "twig/twig": "^2.0",
-        "symfony/yaml": "^3.2 || ^4.0",
-        "symfony/expression-language": "^3.2 || ^4.0",
-        "symfony/framework-bundle": "^3.2 || ^4.0",
-        "symfony/phpunit-bridge": "^3.2 || ^4.0",
+        "symfony/yaml": "^4.0",
+        "symfony/expression-language": "^4.0",
+        "symfony/framework-bundle": "^4.0",
```

Comment:
Can Symfony 5 be added as well?

Reply:
Valid suggestion. I will look into it :+1: